File: BeautySalon/core/src/main/java/dao/entities/Visit.java
package dao.entities;

import dao.entities.people.Client;
import dao.entities.people.Employee;
import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
@RequiredArgsConstructor
public class Visit {
    private Long id;
    private LocalDate visitDate;
    private LocalDateTime startTime;
    private Servize service;
    private Client client;
    private Employee master;

    public Visit(Long id,
                 LocalDate visitDate,
                 LocalDateTime startTime,
                 Servize service,
                 Client client,
                 Employee master) {
        this.id = id;
        this.visitDate = visitDate;
        this.startTime = startTime;
        this.service = service;
        this.client = client;
        this.master = master;
    }
}

File: BeautySalon/angularclient/src/app/model/entities/visit.ts
import {Time} from "@angular/common";
import {Service} from "./service";
import {Client} from "./client";
import {Employee} from "./employee";

export class Visit {

  public id?: number;
  public visitDate?: Date;
  public startTime?: Time;
  public service!: Service;
  public client!: Client;
  public master!: Employee
  ;
}
